centreon/www/install/php/Update-23.10.0.php

```diff
@@ -18,3 +18,40 @@
  * For more information : contact@centreon.com
  *
  */
+
+require_once __DIR__ . '/../../class/centreonLog.class.php';
+$centreonLog = new CentreonLog();
+
+//error specific content
+$versionOfTheUpgrade = 'UPGRADE - 23.10.0: ';
+$errorMessage = '';
+
+$removeNagiosPathImg = function(CentreonDB $pearDB) {
+    $selectStatement = $pearDB->query("SELECT 1 FROM option WHERE `key`='nagios_path_img'");
+    if($selectStatement->rowCount() > 0) {
+        $pearDB->query("DELETE FROM option WHERE `key`='nagios_path_img'");
+    }
+};
+
+try {
+    // Transactional queries
+    $pearDB->beginTransaction();
```

Comment:
If you start a new SQL transaction when another one has already been started, the previous transaction will be automatically committed.
```suggestion
    if (! $pearDB->inTransaction()) {
        $pearDB->beginTransaction();
    }
```